### PR TITLE
Support aggregation layers in CARTO fetchMap

### PR DESCRIPTION
**Pull request by felixpalmer**
For https://github.com/visgl/deck.gl/issues/6760

#### Change List
- Implement binding for `CPUGridLayer`, `HeatmapLayer` and `HexagonLayer` in `fetchMap`
- Tests

### CODE REVIEW

File: modules/carto/src/api/maps-v3-client.ts [hunk 1]
```diff
@@ -338,16 +338,17 @@ async function _fetchMapDataset(
...
   const {url, mapFormat} = await _fetchDataUrl({
+    clientId,
     credentials: {...credentials, accessToken},
     connection,
     columns,
+    format,
     geoColumn,
     source,
-    type,
-    clientId
```
Comment (alasarr) [inline modules/carto/src/api/maps-v3-client.ts:350]: Why is clientId removed? If we do that we'll lose the tracing at metrics

**Review by alasarr — CHANGES_REQUESTED** [review]
Not sure why are we removing clientId

File: modules/carto/src/api/maps-v3-client.ts (hunk 1 above)
Comment (felixpalmer) [reply modules/carto/src/api/maps-v3-client.ts:350]: It's not removed, I just moved it to the top of the params list to have it alphabetical

**Review by alasarr — APPROVED** [review]

File: modules/carto/src/api/maps-v3-client.ts (hunk 1 above)
Comment (alasarr) [reply modules/carto/src/api/maps-v3-client.ts:350]: My bad :)

**Pull request merged**